docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -2,16 +2,22 @@
 Getting Started with Galaxy
 ===========================
 
-Galaxy workflow support in Dockstore is new and a preview of
-more full featured support to come.
+Dockstore supports Galaxy workflows and the :doc:`Launch with Galaxy <../../launch-with/galaxy-launch-with>` tutorial elaborates different ways to launch a Galaxy workflow. Please note that Galaxy workflows cannot be launched by the Dockstore CLI.
+
+Galaxy instances are offered by many resources, free and otherwise. Each come with their own advantages and users can make a choice based on their own needs.
+
+Using Galaxy Servers
+--------------------
+Institutions across the world maintain Galaxy instances that are often shared with the greater community for free, subsidized by grants, such as the `Use Galaxy Org <https://usegalaxy.org/>`. It offers a quick and customizable way to spin up a Galaxy instance. These are great ways to learn and use Galaxy, and many offer valuable learning resources such as the `Galaxy Training Network <https://training.galaxyproject.org/>`.
```

Comment:
"It offers a quick and customizable way to spin up a Galaxy instance." should maybe be "It offers a quick and customizable way to spin up a Galaxy workflow."

Not sure, but would have thought usegalaxy.org is a Galaxy instance itself as opposed to a way to spin up new ones.

Reply:
What I mean to say is that the user can customize their Galaxy "workspace" in the sense of the compute space and resources without having to know much about the cloud backend, and hence it is quick. I am looking for another word for what I want to convey. Changing it to Galaxy workflow makes it seem like the workflow itself is customizable, which it may be, but not what I want to focus on in this section.